src/zookeeper_config.py

```diff
@@ -78,7 +94,7 @@ def set_kafka_opts() -> None:
         opts_string = " ".join(OPTS)
         safe_write_to_file(content=f"KAFKA_OPTS={opts_string}", path="/etc/environment", mode="a")
 
-    def create_properties(self, config):
+    def create_properties(self, config, cluster):
```

Comment:
Although this is obviously fine, could we add a test for this to avoid regression?